src/jit/gentree.h

```diff
@@ -2039,7 +2039,7 @@ struct GenTree
 
     inline bool IsIntegralConst() const;
 
-    inline bool IsIntCnsFitsInI32();
+    inline bool IsIntCnsFitsInI32(); // Constant fits in INT32
```

Comment:
Minor: I don't think this comment adds a lot, but if you add it here, you should probably have a similar comment on the I8 variants below.

Reply:
OK